instrumentserver/apps.py

```diff
@@ -77,15 +76,17 @@ def serverScript() -> None:
                initScript=args.init_script,
                serverConfig=serverConfig,
                stationConfig=stationConfig,
-               pollingThread=pollingThread)
+               pollingThread=pollingThread,
+               ipAddresses=ipAddresses)
     else:
         serverWithGui(port=args.port,
                       addresses=args.listen_at,
                       initScript=args.init_script,
                       serverConfig=serverConfig,
                       stationConfig=stationConfig,
                       guiConfig=guiConfig,
-                      pollingThread=pollingThread)
+                      pollingThread=pollingThread,
+                      ipAddresses=ipAddresses)
```

Comment:
Keep singular term `externalAddress` or something like that throughout the entire class

Reply:
Talked in person